src/averageAges.js

```diff
@@ -15,7 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const menByCentury = people
+    .filter(el => el.sex === 'm' && Math.ceil(el.died / 100) === century);
+
+  const men = people
+    .filter(el => el.sex === 'm');
+
+  const calcAverage = function(someArray) {
+    const BornYear = someArray.reduce((born, obj) => born + obj.born, 0);
+    const DiedYear = someArray.reduce((died, obj) => died + obj.died, 0);
```

Comment:
create more meaningful names
```suggestion
    const sumOfBornYears = people.reduce((bornSum, ({ born })) => bornSum + born, 0);
    const sumOfDiedYears = people.reduce((diedSum, ({ died })) => diedSum + died, 0);
```